components/camview.py

```diff
@@ -15,7 +15,7 @@
 
 class Canvas(CanvasBase):
     # True: use arcball controls, False: use orbit controls
-    arcball_control = True
+    arcball_control = False
```

Comment:
I know we changed this value while we were having a meeting to test if the rotation issue is only for arcball rotation mode. Should we change this back to True?